### PR TITLE
docs: Corrected the manual for Alpine guest setup

### PR DESCRIPTION
**Pull request by ustiugov**
Corrected syntax for `ash` execution in `docs`.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ustiugov** [comment]
hi @aghecenco, it looks like this PR still needs 2 reviews from the team before it could be merged so I am going to re-assign it back to you. If there are any changes I need to do, please let me know.

File: docs/rootfs-and-kernel-setup.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Alpine Linux:
   
    # Then, copy the newly configured system to the rootfs image:
    for d in bin etc lib root sbin usr; do tar c "/$d" | tar x -C /my-rootfs; done
-   mkdir /my-rootfs/{dev,proc,run,sys,var}
```
Comment (petreeftime) [inline docs/rootfs-and-kernel-setup.md]: This bash syntax is valid. However, this is not valid sh syntax. Can you please clarify this in the patch description? Could you also please make the name of the patch more specific?

File: docs/rootfs-and-kernel-setup.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ Alpine Linux:
...
+   for n in dev proc run sys var; do mkdir /my-rootfs/$n; done
```
Comment (petreeftime) [inline docs/rootfs-and-kernel-setup.md]: I suggest renaming `n` to `d` or `dir` or something more specific.

**Comment by ustiugov** [comment]
hi @aghecenco, @petreeftime 

Thank you for the comments, I have applied the requested changes.

**Comment by aghecenco** [comment]
One last thing - the commit message refers to `ash syntax`, is this a typo?

**Comment by ustiugov** [comment]
Apologies, fixed.

**Pull request merged**